content/400-reference/200-api-reference/250-error-reference.mdx

```diff
@@ -525,6 +525,22 @@ Interactive transaction error:
 
 Accelerate-related errors start with `P6xxx`.
 
+#### <inlinecode>P6000</inlinecode> (<inlinecode>ServerError</inlinecode>)
+
+Generic Error to catch all other errors.
+
+#### <inlinecode>P6001</inlinecode> (<inlinecode>InvalidDataSource</inlinecode>)
+
+The URL is malformed; for instance, it does not use the `prisma://` protocol.
+
+#### <inlinecode>P6002</inlinecode> (<inlinecode>Unauthorized</inlinecode>)
+
+The API Key in the connection string is invalid.
+
+#### <inlinecode>P6003</inlinecode> (<inlinecode>PlanLimitReached</inlinecode>)
+
+The included usage of the current plan has been exceeded. This can only occur on the free plan.
```

Comment:
maybe link "free plan" to our pricing page?

Reply:
That's a good idea, I'll do that